server/test/integration/vitalsTest/handleHeartbeatTest.js

```diff
@@ -215,6 +215,17 @@ describe('vitals.js integration tests: handleHeartbeat', () => {
       sandbox.restore()
     })
 
+    it('should log a Sentry alert when the INS sensor value is less or equal to zero', async () => {
+      const response = await chai.request(server).post('/api/heartbeat').send({
+        coreid: radar_coreID,
+        data: `{"isINSZero":true,"doorMissedMsg":0,"doorMissedFrequently":false,"doorLastMessage":-1,"doorLowBatt":-1,"doorTampered":-1,"resetReason":"NONE","states":[]}`,
+        api_key: webhookAPIKey,
+      })
+
+      expect(response).to.have.status(200)
+      expect(helpers.logSentry).to.have.been.calledWith(`INS sensor is less or equal zero at ${testLocation1Id}`)
+    })
+
```

Comment:
This is looking good but I'm wondering if instead of adding your `it` statement to an existing test, you should instead create a new test (starting with the describe statement) at the bottom of the file, so that you are just testing your own functionality. See what Micah had done is his PR that I referenced.

Reply:
I am going to keep it the same since my test statement relates to the same description field as in the previous code, specifically when a heartbeat message is sent. Creating a separate description field is unnecessary and would complicate the overall code, making it harder to track and read.